research/object_detection/dataset_tools/create_ava_tf_record.py

```diff
@@ -0,0 +1,457 @@
+# Copyright 2019 The MediaPipe Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Modified by Kaushik Shivakumar for the AVA Actions Dataset
+# to work without MediaPipe, code started by Bryan Seybold.
+
+r"""Code to download and parse the AVA dataset for TensorFlow models.
+
+The [AVA data set](
+https://research.google.com/ava/index.html)
+is a data set for human action recognition.
+
+This script downloads the annotations and prepares data from similar annotations
+if local video files are available. The video files can be downloaded
```

Comment:
Can you also clarify, whether this script works for both AVA actions and AVA kinetics in the documentation?  What options to pass choosing this etc

Reply:
This script only works for AVA actions, should I work on AVA kinetics as well?

Reply:
I will update the documentation to reflect this.